src/css/style.css

```diff
@@ -10,3 +10,96 @@
   vertical-align: baseline;
 }
 
+
+#root{
+  background-color: #FFDEE9;
+  background-image: linear-gradient(0deg, #FFDEE9 0%, #B5FFFC 100%);
+   min-height: 100vh;
+   overflow: hidden;
+ }
+
+ .app-wrapper{
+  border: 2px solid black;
+  border-radius: 2%;
+  height: 65%;
+  margin: 8em auto;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  width: 700px;
+ }
+
+ .calculator__display{
+  border: 1px solid rgba(0,0,255, 0.1);
+  border-radius: 1.5%;
+  padding: 1rem 0.75rem;
+  width: 98%;
+  height: 100px;
+  margin-top: 0.9%;
+  margin-left: 1%;
+  margin-right: 1%;
+  font-family: 'Press Start 2P', cursive;
+  font-size: 2rem;
+  background-color: grey;
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-end;
+ }
+
+ p{
+  color: white;
+  padding-right: .5rem;
+ }
+
+ .calculator_button{
+   margin-top: 1%;
+ }
```

Comment:
this component should be styled with flex ![shot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png) 
that way you can display the child components in a column direction

Reply:
I used flex. When I say `display: flex` I think you know that is flex right or you are just trying to pull my leg?